src/java_bytecode/jar_file.cpp

```diff
@@ -6,15 +6,13 @@ Author: Daniel Kroening, kroening@kroening.com
 
 \*******************************************************************/
 
+#include <cstring>
 #include <cassert>
+#include <iostream>
```

Comment:
it's not, this was just for debugging / proof of concept, I will add the MIT-licensed "uroni" version and clean this up